### PR TITLE
fix(StatusInput): fixing various layout issues

### PR DESCRIPTION
**Pull request by alexandraB99**
### Checklist

- [x] follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
  - the scope should be the component's name e.g: `feat(StatusListItem): ... `
  - when adding new components, the scope is the module e.g: `feat(StatusQ.Controls): ...`
- [ ] add documentation if necessary (new component, new feature)
- [x] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [x] test changes in both light and dark theme?
- [ ] is this a breaking change?
    - [ ] use the dedicated `BREAKING CHANGE` commit message section
    - [ ] resolve breaking changes in [status-desktop](https://github.com/status-im/status-desktop)
        - [ ] (pre-merge) adapt code to breaking changes
        - [ ] (post-merge) update StatusQ submodule pointer
- [x] test changes in [status-desktop](https://github.com/status-im/status-desktop)

### CODE REVIEW

File: src/StatusQ/Controls/StatusInput.qml [hunk 1]
```diff
@@ -310,24 +335,32 @@ Item {
         root.valid = Object.values(asyncErrors).length == 0
     }
 
-    implicitWidth: inputLayout.implicitWidth
-    implicitHeight: inputLayout.implicitHeight
-    height: implicitHeight
-    width: implicitWidth
+    QtObject {
+        id: internal
+        property int inputHeight: statusBaseInput.multiline || (root.minimumHeight > 0) || (root.maximumHeight > 0)?
```
Comment (caybro) [inline src/StatusQ/Controls/StatusInput.qml:340]: `readonly`

**Review by caybro — APPROVED** [review]
Really nice cleanup!

**Review by noeliaSD — APPROVED** [review]
LGTM!!

File: src/StatusQ/Controls/StatusInput.qml (hunk 1 above)
Comment (alexandraB99) [reply src/StatusQ/Controls/StatusInput.qml]: done

**Pull request merged**